imapclient/client_test.go

```diff
@@ -14,13 +15,27 @@ import (
 const (
 	testUsername = "test-user"
 	testPassword = "test-password"
+
+	simpleEml = `MIME-Version: 1.0
+	Message-Id: <191101702316132@example.com>
+	Content-Transfer-Encoding: 8bit
+	Content-Type: text/lain; charset=utf-8
+	
+	This is my letter!`
 )
 
 func newClientServerPair(t *testing.T, initialState imap.ConnState) (*imapclient.Client, io.Closer) {
 	memServer := imapmemserver.New()
 
 	user := imapmemserver.NewUser(testUsername, testPassword)
 	user.Create("INBOX", nil)
+
+	emlReader := strings.NewReader(simpleEml)
+	_, err := user.Append("INBOX", emlReader, &imap.AppendOptions{})
+	if err != nil {
+		t.Fatalf("Append simpleEml to INBOX = %s", err)
```

Comment:
`%s` is incorrect here, because the value printed has the type `error`.

Reply:
actually it is ok to print errors with %s
s looks if value has interface Stringer or Error
https://cs.opensource.google/go/go/+/refs/tags/go1.21.5:src/fmt/print.go;l=658

So this 2 ways of printing are equal
```
t.Fatalf("Append simpleEml to INBOX = %s", err.Error())
```
```
t.Fatalf("Append simpleEml to INBOX = %s", err)
```

%v does the same , but %s seems like better choice to me since we expect to see a string returned by Error() method, maybe I should switch to using first example with %s and explicit Error call

Reply:
Hm right. But in general I prefer to just use `%v`. `%s` doesn't have any benefit, we don't care about the type here.